example/app.js

```diff
@@ -81,6 +81,10 @@ search.addWidget(
     templates: {
       header: '<div class="panel-heading">Price ranges</div>',
       item: require('./templates/and.html')
+    },
+    transformData: function(data) {
+      data.name = data.name.replace(/(\d+) - (\d+)/, '$$$1 - $$$2').replace(/> (\d+)/, '> $$$1');
```

Comment:
> it doesn't completely copy the resulting data... and not at all for the hierarchical facets

Maybe we should also fix this inside the helper then and ensure we always send # instances to our API consumer?